packages/webapp/src/components/TileDashboard/SummaryTiles/styles.module.scss

```diff
@@ -0,0 +1,81 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+.summaryTiles {
+  display: inline-flex;
+  align-items: flex-start;
+  gap: 4px;
+  align-self: stretch;
+
+  user-select: none;
+  font-family: 'Open Sans';
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: normal;
+}
+
+.totalCount {
+  border-radius: 10px 4px 4px 10px;
+}
+
+.typeCount {
+  border-radius: 4px;
+}
+
+.totalCount,
+.typeCount {
+  height: 100%;
+  display: flex;
+  width: 56px;
+  padding: 8px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: -2px;
+  flex-shrink: 0;
+  text-transform: uppercase;
+
+  background: #0d2926; // not a named colour in Figma
+  box-shadow: 0px 1px 0px 0px var(--Colors-Secondary-Secondary-green-900, #325442);
+
+  .countText {
+    color: var(--Colors-Secondary-Secondary-green-200, #c1e6d2);
+    font-family: 'Open Sans';
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: normal;
+  }
+}
+
+.countLabel {
+  color: var(--Colors-Secondary-Secondary-green-200, #c1e6d2);
+  font-family: 'Open Sans';
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: normal;
+}
+
+.totalCount:hover,
+.typeCount:hover {
+  background: var(--Colors-Neutral-Neutral-900, #2b303a);
+  box-shadow: 0px 1px 0px 0px rgba(0, 0, 0, 0.25);
+
+  .countText {
+    color: var(--Btn-primary-pristine, #ffcf54);
+  }
+}
```

Comment:
Just FYI, I asked Loïc why the colour changes, and it looks like it was leftover from his previous idea, so I will remove these in my PR later.
https://www.figma.com/file/eMffC0h3JPlWtzNttxJpil?node-id=3089:43641&mode=design#710095392